### PR TITLE
AZ-657 Preparation for increase NominatorsRewardedPerValidator to 4096

### PR DESCRIPTION
**Pull request by Marcin-Radecki**
Testing in AWS pending, locally it works.

### CODE REVIEW

File: aleph-client/src/lib.rs [hunk 1]
```diff
@@ -18,8 +18,12 @@ pub use session::{
 pub use staking::{
     bond as staking_bond, force_new_era as staking_force_new_era,
     set_staking_limit as staking_set_staking_limits, validate as staking_validate,
+    wait_for_full_era_completion as staking_wait_for_full_era_completion,
```
Comment (pmikolajczyk41) [inline aleph-client/src/lib.rs:21]: I'm not so convinced to prefixing such methods; Ideally, I would keep these library modules public, but this is against proposed ADR...

**Review by pmikolajczyk41 — APPROVED** [review]
only some minor comments, but in general, looks nice

one remark: when you are done with AWS testing, it would be nice to have somewhere average block occupancy exposed - either here, or in task ticket

**Review by pmikolajczyk41 — CHANGES_REQUESTED** [review]
<del>Now I spot this: please remove e2e dependency from benches/payout-stakers Cargo.toml</del>
I was sure that we get rid of all of them, sorry

**Review by pmikolajczyk41 — APPROVED** [review]

File: aleph-client/src/lib.rs (hunk 1 above)
Comment (timorl) [reply aleph-client/src/lib.rs:21]: I would prefix them only if they are not readable without the prefix – here I would just `pub use` the original name.

Comment (Marcin-Radecki) [reply aleph-client/src/lib.rs:21]: Okay, will do

File: benches/payout-stakers/src/main.rs [hunk 2]
```diff
@@ -1,31 +1,29 @@
...
+    balances_batch_transfer, create_connection, keypair_from_string, staking_bond,
+    staking_validate, staking_wait_for_full_era_completion, staking_wait_for_next_era, Protocol,
 };
+use e2e::staking::{batch_bond, batch_nominate, check_non_zero_payouts_for_era};
 use log::info;
-use primitives::staking::{MIN_NOMINATOR_BOND, MIN_VALIDATOR_BOND};
+use primitives::staking::{
+    MAX_NOMINATORS_REWARDED_PER_VALIDATOR, MIN_NOMINATOR_BOND, MIN_VALIDATOR_BOND,
+};
 use rayon::prelude::*;
```
[the remark this answers is not in the document]
Comment (Marcin-Radecki) [reply benches/payout-stakers/src/main.rs:10]: Right, but here I think It's justified as it is how library handles it: https://docs.rs/rayon/1.5.1/rayon/prelude/index.html

File: benches/payout-stakers/src/main.rs [hunk 3]
```diff
@@ -1,31 +1,29 @@
...
 use std::iter;
 use substrate_api_client::{
-    extrinsic::staking::RewardDestination,
-    {rpc::WsRpcClient, AccountId, Api, XtStatus},
+    extrinsic::staking::RewardDestination, rpc::WsRpcClient, AccountId, Api, XtStatus,
 };
 
 // testcase parameters
-const NOMINATOR_COUNT: u64 = 1024;
+const NOMINATOR_COUNT: u64 = MAX_NOMINATORS_REWARDED_PER_VALIDATOR as u64;
```
[the remark this answers is not in the document]
Comment (Marcin-Radecki) [reply benches/payout-stakers/src/main.rs:19]: Yes, but I wanted to keep type `u32` which is ` pallet_staking::Config::MaxNominatorRewardedPerValidator` type.

[the remark this answers is not in the document]
Comment (pmikolajczyk41) [reply benches/payout-stakers/src/main.rs:19]: you mean `u64`?

**Comment by Marcin-Radecki** [comment]
In this PR we won't change it to 4096, to make this PR more approachable to review.

File: aleph-client/src/lib.rs (hunk 1 above)
Comment (Marcin-Radecki) [reply aleph-client/src/lib.rs]: Done

File: benches/payout-stakers/src/main.rs (hunk 3 above)
[the remark this answers is not in the document]
Comment (Marcin-Radecki) [reply benches/payout-stakers/src/main.rs:19]: Sorry, right, `u64`.

**Pull request merged**